### ISSUE TITLE
Avoid identifying the content provider identity.

### ISSUE THREAD

**Issue by VMatrix1900** [VMatrix1900/draft-quic-structured-connection-id#1]
Sorry, I don't see how that would work. ISP puts the key onto a publicly-available server at, let's say, https://www.t-mobile.com/key.txt, and all the content providers would use that key, and nobody would abuse it? That can't be what you have in mind. I only see a solution with some sort of authorization for Certain Special content providers to use that key. And each content provider would be given its own key because unique keys are necessary for hygiene (key rotation and the eventual fallout of their contract and need to pull their key without disrupting service for other content providers). by @danwing

**Comment by VMatrix1900** [comment]
No. The key should not be public. Authentication is required to obtain the key. The key can be anoymous using the technique developed in  [Privacy Pass](https://datatracker.ietf.org/doc/draft-ietf-privacypass-architecture/). @danwing

**Comment by danwing** [comment]
Ok, thanks!
